Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM ghcr.io/samvera/hyku/base:latest AS hyku-knap-base
+ARG BASE_TAG=${BASE_TAG:-latest}
+FROM ghcr.io/samvera/hyku/base:${BASE_TAG} AS hyku-knap-base
 
 # This is specifically NOT $APP_PATH but the parent directory
 COPY --chown=1001:101 . /app/samvera
-RUN ln -s /app/samvera/bundler.d /app/.bundler.d
-ENV BUNDLE_LOCAL__HYKU_KNAPSACK=/app/samvera
-ENV BUNDLE_DISABLE_LOCAL_BRANCH_CHECK=true
 
-RUN bundle install --jobs "$(nproc)"
+# point bundler-inject at your canonical inject dir (build + runtime)
+ENV BUNDLE_BUNDLER_INJECT__GEM_PATH=/app/samvera/bundler.d
+ENV BUNDLE_DISABLE_LOCAL_BRANCH_CHECK=true
 
+# install gems with a small parallelism cap
+RUN jobs="$(nproc)"; \
```

Comment:
This was something brought over from hykuup_knapsack to help gem install parallelism